drivers/pinmux/stm32/pinmux_stm32f0.h

```diff
@@ -10,91 +10,361 @@
 /**
  * @file Header for STM32F0 pin multiplexing helper
  */
+/* Port A */
+#define STM32F0_PINMUX_FUNC_PA0_USART1_CTS \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUPDR_NO_PULL)
+#define STM32F0_PINMUX_FUNC_PA0_USART2_CTS \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUPDR_NO_PULL)
+#define STM32F0_PINMUX_FUNC_PA0_USART4_TX \
+	(STM32_PINMUX_ALT_FUNC_4 | STM32_PUSHPULL_PULLUP)
 
-#define STM32F0_PINMUX_FUNC_PB6_USART1_TX \
```

Comment:
It's ok to reorganize the file, but please do it in a separate commit, this making it hard to review.

Reply:
I have created a new pull request for this.